examples/compiled/diverging_color_points.vg.json

```diff
@@ -174,7 +174,23 @@
         {
             "stroke": "color",
             "title": "Weight_in_lbs",
-            "type": "gradient"
+            "type": "gradient",
+            "encode": {
+                "symbols": {
```

Comment:
The logic is wrong.  Why would you need to set opacity for both symbols and gradient?